src/main/views/admin-dashboard.njk

```diff
@@ -13,19 +13,58 @@
         <h1 class="govuk-heading-l">{{ heading }}</h1>
     </div>
     <div class="cards-wrapper">
-        {% for card in cards %}
-            <div class="govuk-!-margin-top-2 govuk-!-margin-bottom-2 admin-card">
-                <div class="account-card govuk-!-margin-left-2 govuk-!-margin-right-2">
-                    <p>
-                        <a href="{{ card.link }}"
-                           class="govuk-link govuk-body govuk-!-font-weight-bold"
-                           id="card-{{ card.link }}"
-                        >{{ card.title }}</a>
-                    </p>
-                    <p class="govuk-body">{{ card.description }}</p>
-                </div>
-            </div>
-        {% endfor %}
+      {% if user.isAdmin %}
+      <div class="govuk-!-margin-top-2 govuk-!-margin-bottom-2 admin-card">
+        <div class="account-card govuk-!-margin-left-2 govuk-!-margin-right-2">
+          <p>
+            <a href="{{ uploadCard.link }}"
+               class="govuk-link govuk-body govuk-!-font-weight-bold"
+               id="card-{{ uploadCard.link }}"
+            >{{ uploadCard.title }}</a>
+          </p>
+          <p class="govuk-body">{{ uploadCard.description }}</p>
+        </div>
+      </div>
+      {% endif %}
+      {% if user.isAdmin %}
```

Comment:
It is very minor....guess we don't need a further 'if' condition here as it can be combined with the previous "if"?

Reply:
Good spot :) Now combined with the previous if statement